src/test/java/org/fundacionjala/coding/yerel/StopSpinWordsTest.java

```diff
@@ -0,0 +1,61 @@
+package org.fundacionjala.coding.yerel;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * this is a test of StopSpinWords.
+ */
+public class StopSpinWordsTest {
+
+    /**
+     * this test assert "Hey wollef sroirraw".
+     */
+    @Test
+    public void spinWords() {
+        StopSpinWords stopSpinWords = new StopSpinWords();
```

Comment:
You are repeating this line of code in all the tests
Could you move it to a @Before method